rabit/rabit/Album/AlbumViewController.swift

```diff
@@ -85,37 +86,26 @@ private extension AlbumViewController {
 
     func initializeDataSource() -> AlbumDataSource {
         return AlbumDataSource(
-            configureCell: { dataSource, collectionView, indexPath, item in
-                guard let cell = collectionView.dequeueReusableCell(
-                    withReuseIdentifier: AlbumCell.identifier,
-                    for: indexPath) as? AlbumCell else { return UICollectionViewCell(
-                ) }
-
-                let itemData = dataSource.sectionModels[indexPath.section].items[indexPath.item]
-
-                cell.configure(with: itemData)
-                return cell
-            },
-            configureSupplementaryView: { (dataSource, collectionView, kind, indexPath) in
-                guard kind == UICollectionView.elementKindSectionHeader,
-                      let header = collectionView.dequeueReusableSupplementaryView(
-                          ofKind: kind,
-                          withReuseIdentifier: AlbumHeaderView.identifier,
-                          for: indexPath
-                      ) as? AlbumHeaderView else {
-                    return UICollectionReusableView()
-                }
-                let headerTitle = dataSource.sectionModels[indexPath.section].categoryTitle
-                header.configure(with: headerTitle)
-
-                return header
-            })
+            animationConfiguration: AnimationConfiguration(
+                insertAnimation: .right,
+                reloadAnimation: .fade,
+                deleteAnimation: .left
+            ),
+            configureCell: configureCell,
+            configureSupplementaryView: configureSupplementaryView,
+            canMoveItemAtIndexPath: canMoveItemAtIndexPath
+        )
     }
 
     func bind() {
         guard let viewModel = viewModel else { return }
 
         viewModel.albumData
+            .map { albumArray in
+                albumArray
+                    .map { ($0, $0.items) }
+                    .map(AlbumSectionType.init(model:items:))
```

Comment:
```suggestion
                    .map(AlbumSectionType.init(model:$0, items:$0.items))
```
위처럼 컴파일 가능하면 map 호출을 1번만 해도 괜찮을 것 같아요!
혹시 가독성 때문에 2번 호출한건가요!??

Reply:
넵..ㅎㅎ 가독성 때문에 2번 했습니다!
위의 map은 아래의 map에 있는 메소드 매개변수 쌍 맞춰주려고 했어요..ㅎㅎ

Reply:
코멘트 보고 생각해보니, 불필요하게 map을 2번 사용하면 뭔가 성능상 안좋은게 있긴할것 같네요ㅎㅎ
한 번 사용하는 방향으로 수정하겠습니다!